en-plugin/classes/controller/menu/class-pages-datatable-controller.php

```diff
@@ -1,237 +0,0 @@
-<?php
```

Comment:
Was this unused? Can't find it in the new code.

Reply:
After rechecking on it, found unused. hence removed with all references.